engine/samples/cars/page.md

```diff
@@ -0,0 +1,58 @@
+# Cars {#examples-engine-cars}
+
+@brief Sample that spawns multiple assets and updates them every frame.
+
```

Comment:
Would be nice to include a screenshot of the sample here, so that the reader knows what they are supposed to see.